docs/guides/code-server.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: code-server
+title: Visual Studio Code Server
```

Comment:
I think `code-server` is the correct name for the project now. 

\cc @jsjoeio

Reply:
yes, that's right!

Reply:
Yea, I was mostly using this terminology for those unfamiliar with code-server, who may not immediately realize it's related to VSCode.

Reply:
well you could always put "code-server — VSCode in the browser" but code-server should be used for the name